### ISSUE TITLE
sendWaves json result has 'type' param but sendAsset etc do not

### ISSUE THREAD

**Issue by djpnewton** [PyWaves/PyWaves#131]
sendWaves: https://github.com/PyWaves/PyWaves/blob/a07a3cf11a9b6d3888e45b1a8efa4a03e1e55d18/address.py#L436

sendAsset: https://github.com/PyWaves/PyWaves/blob/a07a3cf11a9b6d3888e45b1a8efa4a03e1e55d18/address.py#L554

This seems like a bug

**Comment by PyWaves** [comment]
Do you get any error?

**Comment by djpnewton** [comment]
yes, I was expecting the result to contain a result field. Without it the result does not conform to the json spec for a waves transaction

**Comment by jansenmarc** [comment]
Please show an example of the error you get, together with the code that generates it.
